ngraph/python/tests/test_onnx/utils/model_importer.py

```diff
@@ -0,0 +1,119 @@
+# ******************************************************************************
+# Copyright 2018-2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ******************************************************************************
+
+from collections import defaultdict
+import glob
+import numpy as np
+import os
+from typing import Dict, List, Optional, Pattern, Set, Text, Type
+
+import onnx
+from onnx import numpy_helper
+import onnx.backend.test
+from onnx.backend.base import Backend
+from onnx.backend.test.case.test_case import TestCase as OnnxTestCase
+from onnx.backend.test.runner import TestItem
+
+
+class ModelImportRunner(onnx.backend.test.BackendTest):
+    def __init__(self, backend, models, parent_module=None):
+        # type: (Type[Backend], List[Dict[str,str]], Optional[str]) -> None
```

Comment:
Please use Python3-style type annotations (not comments).

Reply:
done